### DISCUSSION TITLE
ZEN32 Scene switch missing value from MQTT message

### DISCUSSION THREAD

**Discussion by Matt-PMCT** [zwave-js/zwave-js-ui#1161] [Q&A]
I wanted to check if this was expected behavior or something that should be addressed. I have a ZEN32 Scene Switch in zwavejs2mqtt (4.0.1 / 7.3.0) when I push the button I get an MQTT message like this:
`{"time":1620503427189,"value":3}`
then shortly after that I get another message with no value field representing the switch is idle:
`{"time":1620503427189}`

Home Assistant complains in its error logs because the **value_template** is look for a "value" field, but in the 2nd message for when the switch is not pressed the there is no "value". 

In 2Mqtt the log messages:
```
2021-05-08 20:01:59.474 DEBUG MQTT: Publishing to ZwaveJs700/Office/Office_Scene_Switch/91/0/scene/005: { time: 1620504119474, value: 3 } with options { qos: 1, retain: false }
2021-05-08 20:01:59.475 INFO ZWAVE: Node 3: value notification: 91-0-scene-005 3
2021-05-08 20:02:00.476 DEBUG MQTT: Publishing to ZwaveJs700/Office/Office_Scene_Switch/91/0/scene/005: { time: 1620504120476, value: undefined } with options { qos: 1, retain: false }
```
However the actual MQTT message sent does not seem to contain a value field. 

Thank you very much! Let me know if I should convert this to an issue or if it is expected behavior.

**Comment by Matt-PMCT** [comment]
I think I should also add that a "0" value is passed when the button is pushed only once. So that complicates things because you cannot just say that "undefined" should be a zero :(

**Reply by robertsLando** [reply]
value notification are automatically resetted after some seconds (undefined value is sent). ATM this is just a warning on hass side, what we could do is to add a default value when value is undefined in the template
